src/AppBundle/Repository/PageRepository.php

```diff
@@ -275,14 +275,19 @@ public function getWikidataInfo(Page $page): array
             return [];
         }
 
-        $wikidataId = 'Q'.ltrim($page->getWikidataId(), 'Q');
+        $wikidataId = ltrim($page->getWikidataId(), 'Q');
         $lang = $page->getProject()->getLang();
-
-        $sql = "SELECT term_type AS term, term_text
-                FROM wikidatawiki_p.wb_terms
-                WHERE term_full_entity_id = :wikidataId
-                AND term_type IN ('label', 'description')
-                AND term_language = :lang";
+        $wdp = 'wikidatawiki_p';
+
+        $sql = "SELECT wby_name AS term, wbx_text AS term_text
+                FROM $wdp.wbt_item_terms
+                LEFT JOIN $wdp.wbt_term_in_lang ON wbit_term_in_lang_id = wbtl_id
+                LEFT JOIN $wdp.wbt_type ON wbtl_type_id = wby_id
+                LEFT JOIN $wdp.wbt_text_in_lang ON wbtl_text_in_lang_id = wbxl_id
+                LEFT JOIN $wdp.wbt_text ON wbxl_text_id = wbx_id
```

Comment:
Previous query had no LEFT JOIN's, so dependent callers may or may not expect NULL's.

Reply:
Switching to just join